src/components/ProductsSlider/ProductsSlider.tsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
I see it is almost everywhere, if you really need it would be nice to disable this rule in the linter config file (it is easy, ask in chat if you need help). In another case delete it